thrift/src/test/java/com/linecorp/armeria/it/metrics/PrometheusMetricsIntegrationTest.java

```diff
@@ -0,0 +1,248 @@
+                           .lines()
+                           .map(String::trim)
+                           .filter(line -> line.startsWith("armeria_client_responseBytes_count"))
+                           .filter(line -> line.contains(helloServiceMethod))
+                           .anyMatch(line -> line.endsWith("7.0"))).isTrue();
+        assertThat(new BufferedReader(new StringReader(content))
+                           .lines()
+                           .map(String::trim)
+                           .filter(line -> line.startsWith("armeria_client_requestBytes_count"))
+                           .filter(line -> line.contains(helloServiceMethod))
+                           .anyMatch(line -> line.endsWith("7.0"))).isTrue();
+
+        //Failure count
+        assertThat(new BufferedReader(new StringReader(content))
+                           .lines()
+                           .map(String::trim)
+                           .filter(line -> line.startsWith("armeria_server_failure"))
+                           .filter(line -> line.contains(helloServiceMethod))
+                           .anyMatch(line -> line.endsWith("3.0"))).isTrue();
+        assertThat(new BufferedReader(new StringReader(content))
+                           .lines()
+                           .map(String::trim)
+                           .filter(line -> line.startsWith("armeria_client_failure"))
+                           .filter(line -> line.contains(helloServiceMethod))
+                           .anyMatch(line -> line.endsWith("3.0"))).isTrue();
+
+        //Success count
+        assertThat(new BufferedReader(new StringReader(content))
+                           .lines()
+                           .map(String::trim)
+                           .filter(line -> line.startsWith("armeria_server_success"))
+                           .filter(line -> line.contains(helloServiceMethod))
+                           .anyMatch(line -> line.endsWith("4.0"))).isTrue();
+        assertThat(new BufferedReader(new StringReader(content))
+                           .lines()
+                           .map(String::trim)
+                           .filter(line -> line.startsWith("armeria_client_success"))
+                           .filter(line -> line.contains(helloServiceMethod))
+                           .anyMatch(line -> line.endsWith("4.0"))).isTrue();
+
+        //Active Requests 0
+        assertThat(new BufferedReader(new StringReader(content))
+                           .lines()
+                           .map(String::trim)
+                           .filter(line -> line.startsWith("armeria_server_activeRequests"))
+                           .filter(line -> line.contains(helloServiceMethod))
+                           .anyMatch(line -> line.endsWith("0.0"))).isTrue();
+        assertThat(new BufferedReader(new StringReader(content))
+                           .lines()
+                           .map(String::trim)
```

Comment:
Could be extracted into a method:

```java
private static Stream<String> trimmedLines(String content) {
    return (new BufferedReader(new StringReader(content)).lines().map(String::trim);
}
```

Alternatively, you could keep the trimmed lines in a `List` so it can be reused.